.prettierrc

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Would you mind making a PR against `main` that adds this?

Reply:
Should have included it in the original starter code

Reply:
Yes, but I accidentally pushed an error on the last commit. When I changed the return type from Optional<Book> to Book, I got an error in my controller.

"The method orElseThrow(() -> {}) is undefined for the type BookJava(67108964)"

Can we remove this orElseThrow from the method in the controller?

Reply:
I tested it after removing this and seems to work.

Reply:
Yep! Since the Service guarantees it will always return a value, the call site in the controller can assume it will be there (so no need for "or else")